task3_1/src/main/java/electricalAppliances/cleaningDevice/dry/vacuumCleaner/VacuumCleaner.java

```diff
@@ -16,7 +17,12 @@ public VacuumCleaner(String company, String model, int releaseDate, int wattsPow
 
         this.dustCollector = dustCollector;
         this.vacuumCleanerPower = vacuumCleanerPower;
-        this.suctionPower = suctionPower;
+        try {
+            this.suctionPower = suctionPower;
+        } catch (Exception ex) {
```

Comment:
We should use more specific exception. (E.g.: IOException or some custom exception)